### DISCUSSION TITLE
Can I add some filter words, in addition to relevant/irrelevant articles, to refine search results?

### DISCUSSION THREAD

**Discussion by MohsenAskar** [asreview/asreview#838] [Ideas]
Hi all,
Is it possible to use some filter words along with selecting relevant/irrelevant articles to help algorithms to prioritize relevant searches? If not, would it be useful to add such an option?

Thanks

**Comment by jteijema** [comment]
Hi @MohsenAskar,

The highlighting of words is something we're discussing internally, as can be very useful during screening. It allows for quickly being able to identity the critical parts of a record. A drawback might be introducing bias for certain terms, and if the screening is heavily influenced by certain terms, then shouldn't these terms have been included/excluded during the preprocessing phase?

Currently in my personal to-do list is a (blog) write-up on word highlighting, and using extensions in firefox or chrome like [Multi Highlight](https://chrome.google.com/webstore/detail/multi-highlight-search-fi/ghinpogonpmlphfkmnjfapdaoaainejn) for example. Such an extension might be a solution for your problem!

Finally, another github discussion thread on this topic can be found at #482. Whether or not this feature is going to be implemented has yet to be seen 😄.

Thank you,
Jelle

**Reply by MohsenAskar** [reply]
Thanks! :)
